### PR TITLE
feat: CRUD profile module + entity

### PR DESCRIPTION
**Pull request by Mabelzzz**

### CODE REVIEW

File: backend/src/features/profiles/profiles.service.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@nestjs/common';
+import { InjectConnection, InjectModel } from '@nestjs/mongoose';
+import { Connection, Model } from 'mongoose';
+import { COLLECTION_NAME } from 'src/configs/mongoose.config';
```
Comment (viruskizz) [inline backend/src/features/profiles/profiles.service.ts:4]: Dont use root path for import.
Please change to relative

File: mongodb/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# For local development only, no password required
```
Comment (viruskizz) [inline mongodb/docker-compose.yml:1]: No need this file.
We use mongodb atlas on cloud

File: backend/src/app.module.ts [hunk 3]
```diff
@@ -4,14 +4,17 @@ import { AppService } from './app.service';
...
+import { ProfilesModule } from './features/profiles/profiles.module';
+import { AbcModule } from './features/abc/abc.module';
 
 @Module({
   imports: [
     ConfigModule.forRoot(),
     mongooseConnection.root,
     QuizesModule,
+    ProfilesModule,
+    AbcModule,
```
Comment (viruskizz) [inline backend/src/app.module.ts:16]: What is AbcModule ?

File: backend/src/features/profiles/profiles.service.ts (hunk 1 above)
Comment (Mabelzzz) [reply backend/src/features/profiles/profiles.service.ts:4]: ### same as your Quizzes module

image

File: backend/src/app.module.ts (hunk 3 above)
Comment (Mabelzzz) [reply backend/src/app.module.ts]: fixed

File: mongodb/docker-compose.yml (hunk 2 above)
Comment (Mabelzzz) [reply mongodb/docker-compose.yml]: fixed

File: backend/src/features/profiles/profiles.service.ts (hunk 1 above)
Comment (viruskizz) [reply backend/src/features/profiles/profiles.service.ts:4]: Thank you. I will update my component

**Comment by viruskizz** [comment]
We will fix path in next PR.

**Review by viruskizz — APPROVED** [review]

**Pull request merged**